src/clients/api/queries/getVTokenApySimulations/__tests__/__snapshots__/index.spec.ts.snap

```diff
@@ -4,503 +4,503 @@ exports[`api/queries/getVTokenApySimulations > returns the APY simulations in th
 {
   "apySimulations": [
     {
-      "borrowApyPercentage": 1.0483200000054798e-9,
-      "supplyApyPercentage": 2.096640000021919e-9,
+      "borrowApyPercentage": 1.05072e-9,
+      "supplyApyPercentage": 2.09335e-9,
```

Comment:
This might be minor than the infinity values, but is the precision loss acceptable here?

Reply:
I'd say it is. If you check this snapshot: https://github.com/VenusProtocol/venus-protocol-interface/pull/1591/files/adb1abee8898455248302b33dd335d07effcdbf7#diff-6bc877593c72b804d11cf819f5dc5df18ede9281ed7bdb59e1d4e044b4e3df29R8
We only lose precision after the 11th decimals. I think that's fine for a frontend dApp.